warehouse/templates/manage/account/publishing.html

```diff
@@ -25,7 +25,7 @@
 {% macro github_form(request, pending_github_publisher_form) %}
   <p>
     {% trans href="https://docs.github.com/en/actions/deployment/security-hardening-your-deployments/about-security-hardening-with-openid-connect" %}
-    Read more about GitHub Actions's OpenID Connect support <a href="{{ href }}">here</a>.
+    Read more about GitHub Action's OpenID Connect support <a href="{{ href }}">here</a>.
```

Comment:
This was probably my fault originally 🙂 

```suggestion
    Read more about GitHub Actions' OpenID Connect support <a href="{{ href }}">here</a>.
```

Reply:
Naw, I was mucking around, copy and pasting stuff at the end of the day yesterday and was going to come back to this mistake.  Thanks for the change suggestion!